nano/node/confirming_set.cpp

```diff
@@ -255,6 +255,7 @@ nano::container_info nano::confirming_set::container_info () const
 
 	nano::container_info info;
 	info.put ("set", set);
-	info.add ("notification_workers", notification_workers.container_info ());
+	info.put ("notifications", workers.queued_tasks ());
```

Comment:
Is this already added with workers.container_info() below?

Reply:
This might be redundant, right